pkgs/top-level/all-packages.nix

```
@@ -5694,6 +5694,8 @@ with pkgs;
 
   hyprshade = python311Packages.callPackage ../applications/window-managers/hyprwm/hyprshade { };
 
+  hyprlandPlugins = recurseIntoAttrs (callPackage ../application/window-managers/hyprwm/hyprland/plugins.nix { });
```

```suggestion
  hyprlandPlugins = recurseIntoAttrs (callPackage ../applications/window-managers/hyprwm/hyprland/plugins.nix { });
```

see https://gist.github.com/GrahamcOfBorg/f430b5f70beffcd850e48764cd2efbad#file-output-path-comparison-L141